test/cases/chunks/inline-options/index.js

```diff
@@ -81,54 +89,141 @@ it("should be able to use weak mode (without context)", function() {
 				throw new Error("Unexcepted test data");
 		}
 	}
-	require("./dir9/a") // chunks served manually by the user
-	require("./dir9/b")
-	require("./dir9/c")
+	require("./dir9/a"); // chunks served manually by the user
+	require("./dir9/b");
+	require("./dir9/c");
 	return testChunkLoading(load, true, true);
 });
 
-it("should not find module when mode is weak and chunk not served elsewhere", function() {
+it("should not find module when mode is weak and chunk not served elsewhere", function () {
 	var name = "a";
-	return import(/* webpackMode: "weak" */ "./dir10/" + name)
-		.catch(function(e) {
-			expect(e).toMatchObject({ message: /not available/, code: /MODULE_NOT_FOUND/ });
+	return import(/* webpackMode: "weak" */ "./dir10/" + name).catch(function (
+		e
+	) {
+		expect(e).toMatchObject({
+			message: /not available/,
+			code: /MODULE_NOT_FOUND/
 		});
+	});
+});
+
+it("should not find module when mode is weak and chunk not served elsewhere (without context)", function () {
+	return import(/* webpackMode: "weak" */ "./dir11/a").catch(function (e) {
+		expect(e).toMatchObject({
+			message: /not available/,
+			code: /MODULE_NOT_FOUND/
+		});
+	});
+});
+
+it("should contain only one export from webpackExports from module", function () {
+	return import(/* webpackExports: "a" */ "./dir12/a").then(module => {
+		expect(module).toHaveProperty("a");
+		expect(module).not.toHaveProperty("b");
+		expect(module).not.toHaveProperty("default");
+	});
+});
+
+it("should contain only webpackExports from module", function () {
+	return import(/* webpackExports: ["a", "b"] */ "./dir12/b").then(module => {
+		expect(module).toHaveProperty("a");
+		expect(module).toHaveProperty("b");
+		expect(module).not.toHaveProperty("default");
+	});
+});
+
+it("should contain only webpackExports from module in eager mode", function () {
+	return import(
+		/*
+			webpackMode: "eager",
+			webpackExports: ["a", "b"]
+		*/ "./dir12/c"
+	).then(module => {
+		expect(module).toHaveProperty("a");
+		expect(module).toHaveProperty("b");
+		expect(module).not.toHaveProperty("default");
+	});
 });
 
-it("should not find module when mode is weak and chunk not served elsewhere (without context)", function() {
-	return import(/* webpackMode: "weak" */ "./dir11/a")
-		.catch(function(e) {
-			expect(e).toMatchObject({ message: /not available/, code: /MODULE_NOT_FOUND/ });
+it("should contain only webpackExports from module in weak mode", function () {
+	return import(/*
+			webpackMode: "weak",
+			webpackExports: ["a", "b"]
+		*/ "./dir12/d")
+		.then(module => {
+			expect(module).toHaveProperty("a");
+			expect(module).toHaveProperty("b");
+			expect(module).not.toHaveProperty("default");
+		})
+		.catch(function (e) {
+			expect(e).toMatchObject({
+				message: /not available/,
```

Comment:
There should not be an error, otherwise your checks above didn't run. Make sure the module is available, e.g. by importing something from it.